src/main/java/com/microsoft/sqlserver/jdbc/KeyVaultHttpPipelineBuilder.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.sqlserver.jdbc;
+
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpLoggingPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.policy.HttpPolicyProviders;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.http.policy.UserAgentPolicy;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.logging.ClientLogger;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+
+final class KeyVaultHttpPipelineBuilder {
```

Comment:
The methods in this class should probably not be public.

Reply:
I will change the visibility of these methods to package-private.